index.html

```diff
@@ -169,55 +168,55 @@ <h2 id="about-me" class="aboutme_head">About me</h2>
         work and experience! If you like what you see and have a project you need coded, don’t hesitate to
         contact me.</p>
       <a class="web_button center_button" href="#">Get My Resume</a>
-    </div>
-    </div>
-    <div>
-      <ul class="flex_container_secondary">
-        <li class="nested_flexbox">
-          <h3>Languages</h3>
-          <img class="center" src="assets/images/icon-languages.svg" alt="Languages icon">
-          <ul>
-            <li>JavaScript</li>
-            <li>Ruby</li>
-            <li>HTML</li>
-            <li>CSS</li>
-          </ul>
-        </li>
-        <li class="nested_flexbox">
-          <h3>Frameworks</h3>
-          <img class="center ordering" src="assets/images/icon-frameworks.svg" alt="Frameworks icon">
-          <ul>
-            <li>Bootstrap</li>
-            <li>Ruby on rails</li>
-            <li>RSpec</li>
-            <li>Capybara</li>
-            <li>Selenium</li>
-          </ul>
-        </li>
-        <li class="nested_flexbox">
-          <h3>Skills</h3>
-          <img class="center" src="assets/images/icon-skills.svg" alt="Skills icon">
-          <ul>
-            <li>Codekit</li>
-            <li>GitHub</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
-          </ul>
-        </li>
-      </ul>
+      </div>
+  </div>
+  <div>
+    <ul class="flex_container_secondary">
+      <li class="nested_flexbox">
+        <h3>Languages</h3>
+        <img class="center" src="assets/images/icon-languages.svg" alt="Languages icon">
+        <ul>
+          <li>JavaScript</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+      </li>
+      <li class="nested_flexbox">
+        <h3>Frameworks</h3>
+        <img class="center ordering" src="assets/images/icon-frameworks.svg" alt="Frameworks icon">
+        <ul>
+          <li>Bootstrap</li>
+          <li>Ruby on rails</li>
+          <li>RSpec</li>
+          <li>Capybara</li>
+          <li>Selenium</li>
+        </ul>
+      </li>
+      <li class="nested_flexbox">
+        <h3>Skills</h3>
+        <img class="center" src="assets/images/icon-skills.svg" alt="Skills icon">
+        <ul>
+          <li>Codekit</li>
+          <li>GitHub</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </li>
+    </ul>
   </section>
-
+  
   <section>
     <footer>
       <h3 id="contact-info" class="contact_head">I'm always interested in hearing about new projects, so if you'd like
         to chat please get in touch.</h3>
-      <form novalidate action="https://formspree.io/f/meqdykek" method="post">
+      <form novalidate action="https://formspree.io/f/meqnjpgd" method="post">
```

Comment:
- [ ] :recycle: Kindly make sure the project stick to the figma design, i suggest to align the form items center:

  - Your design:
![midYou](https://user-images.githubusercontent.com/97834160/206048613-a318f49a-ffd4-4074-8d5a-1940ed3238ae.png)

  - Figma:
![Formfig](https://user-images.githubusercontent.com/97834160/206048633-050c9f62-7dd3-46e9-a353-e479efa2aecf.png)